tests/compatibility-tests/src/main/resources/servers/hornetqServer.groovy

```diff
@@ -19,7 +23,7 @@ import org.hornetq.core.config.impl.ConfigurationImpl
 import org.hornetq.core.remoting.impl.netty.NettyAcceptorFactory
 import org.hornetq.core.remoting.impl.netty.TransportConstants
 import org.hornetq.jms.server.config.impl.JMSConfigurationImpl
-import org.hornetq.jms.server.config.impl.JMSQueueConfigurationImpl
+import org.hornetq.jms.server.config.impl.*
```

Comment:
nit: but can this expanded import, especially as still explicitly importing other class's from the same package.

Reply:
I didn't care much as this is a groovy script. but i will expand it.

Reply:
I will keep this one as is actually... I have no help from the IDE on what classes to import as this is hornetq... (It wasn't part of my change now anyways).


(this is the reason I'm using groovy.. I can tweak the classloader.. and have no direct dependency on the project.. totally optional. even the scan is done by a plugin.